CONTRIBUTING.md

```diff
@@ -150,6 +150,12 @@ $ ./configure && make -j8 test
 Make sure the linter is happy and that all tests pass. Please, do not submit
 patches that fail either check.
 
+You can check if the linter is happy by running:
```

Comment:
@addaleax Oy! I think I understand you now! I just though you didn't see this addition as useful. Sorry for that.

I will add your proposal right now.